build/Makefile

```diff
@@ -16,7 +16,7 @@
 ##F_MASTER =
 
 # Define the Fortran Compiler. If you are using gfortran, then this needs
-# to be version 4.8 or higher. This variable is simply used to select the right
+# to be version 6 or higher. This variable is simply used to select the right
```

Comment:
Why does this keep showing up? This has nothing to do with this PR?

Reply:
I merged master into develop, the two branches are not synced up.